backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,49 @@
+## Port
+server.port = 5000
+endpoints.env.enabled=true
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
+## MySQL
+spring.datasource.username = ${SECRETS_DB_USERNAME}
+#spring.datasource.username = omegaadmin@omegadatabase3
+spring.datasource.password = ${SECRETS_DB_PASSWORD}
+#spring.datasource.password = omegaDatabase3
+#spring.datasource.username=omegadatabase@omegadatabase3
+#spring.datasource.password=omegaDatabase3
+
+## Hibernate
+spring.jpa.hibernate.ddl-auto = update
+
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
+#spring.jpa.show-sql=true
+
+#spring.datasource.url=jdbc:mysql://omegadatabase3.mysql.database.azure.com/omegarestapi?useLegacyDatetimeCode=false&serverTimezone=UTC
+spring.datasource.url=jdbc:mysql://${SECRETS_DB_URL}?useLegacyDatetimeCode=false&serverTimezone=UTC
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+
+jwt.secret="a@a.com"
+
+#kubectl create secret generic dev-db-secret --from-literal=SECRETS_DB_USERNAME=omegaadmin@omegadatabase3 --from-literal=SECRETS_DB_PASSWORD=omegaDatabase3 --from-literal=SECRETS_DB_URL=jdbc:mysql://omegadatabase3.mysql.database.azure.com/omegarestapi?useLegacyDatetimeCode=false&serverTimezone=UTC
+
+
+#
+#
+## SENDGRID (SendGridAutoConfiguration)
+spring.sendgrid.api-key= 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
```

Comment:
Is this something we should/can publicise?

Reply:
No, this is related to another module (email service), ill remove it and make a secret for it when we need it

Reply:
removed